README.md

```diff
@@ -1 +1,106 @@
-# Nextcloud-OpenProject-App repository
\ No newline at end of file
+# OpenProject as Nextcloud's External App
+
+## Manual Installation
+For the manual installation of `OpenProject` as an external application of `Nextcloud`, make sure that your `Nextcloud` as well as `OpenProject` instance is up and running.
+
+### 1. Install `app_api` application
+
+Assuming you’re in the apps folder directory:
+
+- Clone
+	```bash
+	git clone https://github.com/cloud-py-api/app_api.git
+	```
+- build
+   ```bash
+   cd app_api
+   npm ci && npm run dev
+    ```
+- Enable the `app_api`
+	```bash
+	# Assuming you’re in nextcloud server root directory
+	sudo -u www-data php occ a:e app_api
+	```
+ 
+### 2. Register deploy daemons (In Nextcloud)
+
+- Navigate to `Administration Settings > AppAPI`
+- Click register daemons
+- Select `Manual Install` for Daemon configuration template
+- Put `manual_install` for name and display name
+- Deployment method as `manual-install`
+- Daemon host as `localhost`
+- Click Register
+
+### 3. Running OpenProject locally
+Set up and build `OpenProject` locally following [OpenProject Development Setup](https://www.openproject.org/docs/development/development-environment/)
+After the setup, run `OpenProject` locally with the given command line.
+
+>NOTE: while running the below bash command line for `NC_SUB_FOLDER` put the sub folder path if you are running nextcloud in a sub folder else do remove it.
+
+```bash
+OPENPROJECT_AUTHENTICATION_GLOBAL__BASIC__AUTH_USER=apiadmin \
+OPENPROJECT_AUTHENTICATION_GLOBAL__BASIC__AUTH_PASSWORD=apiadmin \
+OPENPROJECT_RAILS__RELATIVE__URL__ROOT=/<NC_SUB_FOLDER>/index.php/apps/app_api/proxy/openproject-nextcloud-app \
+OPENPROJECT_2FA_DISABLED=false \
+foreman start -f Procfile.dev
+```
+
+### 4. Configure and Run External `openproject-nextcloud-app` application
+Assuming you’re in the apps folder directory:
+
+- Clone
+  ```bash
+  git clone https://github.com/JankariTech/openproject-nextcloud-app.git
+  ```
+- Configure script before running external app
+   ```bash
+	 cd openproject-nextcloud-app
+	 cp ex_app_run_script.sh.example ex_app_run_script.sh
+    ```
+  Once you have copied the script to run the external application, configure the following environments
+
+  - APP_ID is the application id of the external app
+  - APP_PORT is port for the external app
+  - APP_HOST is the host for the external app
+  - APP_SECRET is the secret required for the communication between external app and nextcloud
+  - APP_VERSION is the version of external app
+  - AA_VERSION is the app_api version used
+  - EX_APP_VERSION is the version of external app
+  - EX_APP_ID is the application id of the external app
```

Comment:
```suggestion
  - APP_VERSION is the version of external app
  - AA_VERSION is the app_api version used
  - EX_APP_VERSION is the version of external app
  - EX_APP_ID is the application id of the external app
```

`APP_VERSION` & `EX_APP_VERSION` both have the same description

Reply:
i will once again check what is required and what not required

Reply:
All of them are required so no change.